src/styled/components/Utils.js

```diff
@@ -0,0 +1,17 @@
+import { css } from "styled-components";
+
+export const Utils = {
```

Comment:
@hudakdidit this is pretty cool! This constant allows us to define utils in a similar manner to suit-css without the bulky classes. 

Can be added to whatever element `${ Utils.margin };` then used at the component level `<ButtonGroup mr={-16}>`